### PR TITLE
merge -cp to -dev

### PR DESCRIPTION
**Pull request by luck3y**
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -2,17 +2,18 @@ schema_version: 1
 
 name: "jboss-eap-7/eap71"
 description: "Red Hat JBoss Enterprise Application Platform 7.1 container image"
-version: "7.1.3"
+version: "7.1.4"
+>>>>>>> upstream/eap71-cp
```
Comment (apintea) [inline image.yaml]: Ah, error :)

File: cp-overrides.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 schema_version: 1
+
```
Comment (apintea) [inline cp-overrides.yaml:2]: Unnecessary, but OK :)

File: image.yaml (hunk 1 above)
Comment (luck3y) [reply image.yaml]: Oopsie :)

File: cp-overrides.yaml (hunk 2 above)
Comment (luck3y) [reply cp-overrides.yaml:2]: yeah, it was in -dev, so rather than just deleted it I merged them.

**Review by apintea — APPROVED** [review]

**Pull request merged**